src/codegen/abi.js

```diff
@@ -334,7 +334,8 @@ module.exports = class AbiCodeGenerator {
       member =>
         member.get('type') === 'function' &&
         member.get('outputs', immutable.List()).size !== 0 &&
-        allowedMutability.indexOf(member.get('stateMutability')) !== -1,
+        (allowedMutability.indexOf(member.get('stateMutability')) >= 0 ||
```

Comment:
Isn't [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) more elegant than `indexOf`?

Reply:
You're right, I've updated the check.